src/main/webapp/script.js

```diff
@@ -26,3 +26,34 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+function storeZipCode(){
+    event.preventDefault();
+    var zipcode = document.getElementById("zipcode").value;
+    localStorage.setItem("zipcode", zipcode);
+    window.location.href = "/repList.html";
+}
+
+function getRepList(){
+    var zipcode = localStorage.getItem("zipcode");
+    fetch(`/rep_list?zipcode=${zipcode}`).then(response => response.json()).then((representatives) => {
+        representatives = JSON.parse(representatives);
+        var representativeList = document.getElementById("repList");
+        var offices = representatives.offices;
+        var officials = representatives.officials;
+        for (var i = 0; i < offices.length; i++) {
+            for (number of offices[i]["officialIndices"]){
+                console.log("i: " + i + " number: " + number + " array: " 
+                + offices[i]["officialIndices"]);
```

Comment:
nit: add an additional tab (or as many spaces) to this since its a continuation of the previous line.